Makefile

```diff
@@ -33,5 +33,4 @@ site-docs: site
 	rm -f site/manual/*.bak
 	git -C site add 'manual/gh*.md'
 	git -C site commit -m 'update help docs'
-	git -C site push
```

Comment:
i'm confused why we'd commit and not push?

Reply:
Mainly because a separate commit will be needed to bump the version number in `site/index.html` on release. Pushing before that would rebuild the site with new docs but with old version number referenced on the home page.

Reply:
ah right.